packages/collector/test/test_util/CollectingLogger.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * (c) Copyright IBM Corp. 2022
+ */
+
+'use strict';
+
+const { format } = require('util');
+
+/**
+ * A logger that does not emit logs anywhere but collects them. Useful for validating what has been logged in tests.
+ */
+module.exports = class CollectingLogger {
```

Comment:
This is basically a vehicle to make "Cannot start an AWS SQS entry span when another span is already active." is not logged, that is, we do not run into a situation where the previous SQS entry span is still active.